_pages/Yourself_info.html

```diff
@@ -89,7 +89,7 @@ <h2 id="page_header">Tell us about yourself</h2>
         <div class="alert alert-warning d-none js-medications-alert row" role="alert">
             <i class="fas fa-exclamation-triangle fa-2x fa-fw col-md-1"></i>
             <div class="col-md-11 alert-body">
-              <p>You may continue with the application, but if you cannot retrieve medications you should consider only short term foster periods of two weeks or less.</p>
+              <p>You may continue with the application, but if you cannot go get medications, you should consider only doing short term foster periods of two weeks or less.</p>
```

Comment:
Putting this here bc I can't see how to put it on line 73 - should we change 'preventative' to 'prevention' or something else simpler?